### ISSUE TITLE
Missing media files

### ISSUE THREAD

**Issue by mistermatze** [lmccart/gswp5.js-code#4]
The following files are missing in the media folder:

- SourceCodePro-Regular.ttf which is used in Ex_07_11.js
- robot3.svg which is used in Robot05_Media.js
- alpine.png which is used in Robot05_Media.js
- ortiz.csv which is used in Ex_12_01.js
- cities.csv which is used in Ex_12_03.js
- film.json which is used in Ex_12_04.js
- films.json which is used in Ex_12_05.js
- cincinnati.json which is used in Ex_12_06.js
- blip.wav which is used in Ex_13_01.js

**Comment by mnoblennow** [comment]
I have been searching for these files all over the web and no luck. Does anybody have them?

**Comment by mnoblennow** [comment]
Files found in another Processing book:
www.processing.org/learning/books/media.zip

**Comment by jeremydouglass** [comment]
Seems like the only one not included in that zip is alpine.png. @mnoblennow -- did you find it elsewhere?

**Comment by mnoblennow** [comment]
No, just the file link from Processing website. But, a huge help!

**Comment by jeremydouglass** [comment]
Got it.

https://github.com/processing/processing-docs/blob/master/content/gswp_2e/examples/Robots/Robot05_Media/data/alpine.png
